src/components/table/base-table/base-table.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { CellMeasurer, CellMeasurerCache, MultiGrid } from 'react-virtualized';
 import sortBy from 'lodash/sortBy';
-import getScrollbarSize from 'dom-helpers/scrollbarSize';
+import getScrollbarSize from 'dom-helpers/util/scrollbarSize';
```

Comment:
Was this a bug fix? 🤔

Reply:
I'm confused about this. There must be some weird resolution happening, because when I open `node_modules` on master, the file is at `scrollbarSide`, yet when I open node modules here, it's at `utils/scrollbarSize` and you get a lint warning 🤔